app/models/content_item.rb

```diff
@@ -45,7 +45,11 @@ class ContentItem < ActiveRecord::Base
   validates :details, well_formed_content_types: { must_include: "text/html" }
 
   def requires_base_path?
-    EMPTY_BASE_PATH_FORMATS.exclude?(document_type)
+    EMPTY_BASE_PATH_FORMATS.exclude?(document_type || format)
+  end
+
+  def pathless?
+    !self.requires_base_path? && !Location.exists?(content_item: self)
```

Comment:
I am a bit concerned with slipping knowledge of paths/locations into content_item here. it seems more convenient, but it goes against the general theme that this was originally trying to achieve: https://github.com/alphagov/publishing-api/blob/master/doc/model.md#high-level-diagram

I think this would be a good place to be moved into the WebContentItem, or something similar. 

What do you think?